diesel/src/insertable.rs

```diff
@@ -13,7 +13,7 @@ use sqlite::Sqlite;
 ///
 /// ### Deriving
 ///
-/// This trait can be automatically derived using `diesel_codegen` by adding
+/// This trait can be automatically derived using `diesel_derives` by adding
```

Comment:
If this is automatically included by diesel now, we should remove the

> using `diesel_derives`

same below in all other instances of this text